### PR TITLE
[Bug]: Search Bar Design Is Broken

### LINKED ISSUE #366: 🐞 [Bug]: Search Bar Design Is Broken
### Contact Details

hetpatel2312.hp@gmail.com

### What happened?

I found in the Documentation section bottom right side search field design is broken.
Image:
image

### Relevant log output

_No response_

### 🙏 Would you be willing to submit a PR?

Yes

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by developerHet**
## Change description

> Fix the search bar border in the documentation section.

Before:
image

After:
image

## Type of change
- [x] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

Fix #366 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows project security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to issue tracker where applicable

### CODE REVIEW

**Comment by developerHet** [comment]
@olivierodo can you help me to figure out what's gone wrong?

File: packages/report-ui/vite.config.js.timestamp-1679382265269.mjs
[the remark this answers is not in the document]
Comment (developerHet) [reply packages/report-ui/vite.config.js.timestamp-1679382265269.mjs:1]: No.

[the remark this answers is not in the document]
Comment (developerHet) [reply packages/report-ui/vite.config.js.timestamp-1679382265269.mjs:1]: packages/report-ui/public/styles/element-plus.css
I made changes on this path only.

**Comment by olivierodo** [comment]
I run the Test again. Sounds good now.

File: packages/report-ui/vite.config.js.timestamp-1679382265269.mjs
[the remark this answers is not in the document]
Comment (olivierodo) [reply packages/report-ui/vite.config.js.timestamp-1679382265269.mjs:1]: Ok so can your remove it ?

[the remark this answers is not in the document]
Comment (developerHet) [reply packages/report-ui/vite.config.js.timestamp-1679382265269.mjs:1]: ok i deleted that file.

**Comment by olivierodo** [comment]
Thank you @developerHet for your contribution 🎉.
This fix will be added to the next release 1.0.0 🚀

**Pull request merged**